changelog/undistributed/template.rst

```diff
@@ -1,21 +0,0 @@
---------------------------------------------------------------------------------
```

Comment:
Don't touch this file, it will cause a merge conflict.

Reply:
File restored via git reset